internal/stringify/fmt_test.go

```diff
@@ -88,6 +107,16 @@ func TestFmt(t *testing.T) {
 			input:  []any{1, "a", true, aStringer{"smarf"}},
 			expect: []string{"1", "a", "true", "smarf"},
 		},
+		{
+			name:   "ptr value",
+			input:  []any{ptrString},
+			expect: []string{fmt.Sprintf("<*>(%p)%s", ptrString, ptrTestString)},
+		},
+		{
+			name:   "ptr struct",
+			input:  []any{testStruct},
+			expect: []string{fmt.Sprintf("<*>(%p){ptrV1:<*>(%p)%s inner:<*>(%p){ptrV2:<*>(%p)%s}}", testStruct, ptrString, ptrTestString, testStruct.inner, ptrString, ptrTestString)},
```

Comment:
nit for style standards: height is preferred over length.
```suggestion
			expect: []string{fmt.Sprintf(
			  "<*>(%p){ptrV1:<*>(%p)%s inner:<*>(%p){ptrV2:<*>(%p)%s}}",
			  testStruct,
			  ptrString,
			  ptrTestString,
			  testStruct.inner,
			  ptrString,
			  ptrTestString,
			)},
```